packages/client/src/links/localLink.ts

```
@@ -0,0 +1,243 @@
+            path: op.path,
+            getRawInput: async () => input,
+            ctx,
+            type: op.type,
+            signal,
+          });
+        };
+        run(async () => {
+          switch (op.type) {
+            case 'query':
+            case 'mutation':
+              const result = await runProcedure(op.input);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)